pkg/common/plugin/exec.go

```diff
@@ -96,10 +99,49 @@ func (e *DefaultExecutor) ExecutePlugin(ctx context.Context, pluginPath string,
 	// stdout and stderr for some reason. Ignore failures as stderr is
 	// only informational.
 	if e.Stderr != nil && stderr.Len() > 0 {
-		_, _ = stderr.WriteTo(e.Stderr)
+		scanner := bufio.NewScanner(stderr)
+		for scanner.Scan() {
+			line := scanner.Text()
+
+			switch {
+			case strings.HasPrefix(strings.ToLower(line), "info"):
+				line = strings.Replace(line, "info: ", "", -1)
+				logrus.Infof("[Plugin] %s", line)
```

Comment:
Suggestion: Instead of using the `logrus` package directly, can we consider using the ratify internal context logger found [here](https://github.com/deislabs/ratify/blob/c75c5b4cbe3ab6c434827975514acc954dfd32d4/internal/logger/logger.go#L91). Basically, this is an implementation that uses the `context` passed down to store extra information like the correlationid. and the `GetLogger` method allows the component to be specified making it match the format of other logs. The correlationid is particular helpful to correlate which plugin logs correspond to which http request. You can follow an example [here](https://github.com/deislabs/ratify/blob/c75c5b4cbe3ab6c434827975514acc954dfd32d4/pkg/verifier/notation/notation.go#L80)of how to fetch the logger and use it

Reply:
cc: @binbin-li

Reply:
I see, so I could pass the context that's passed to the `ExecutePlugin` function to the internal logger to provide that additional information? I'll start working on replace logrus with this.

Reply:
QQ: Should I add another compontent type for plugin, or use the [executor](https://github.com/deislabs/ratify/blob/c75c5b4cbe3ab6c434827975514acc954dfd32d4/internal/logger/logger.go#L54) type for the logopt?

Reply:
I think you can introduce a new component type called `plugin` or something similar